source/branding.html.erb

```diff
@@ -21,27 +24,37 @@ title: Brand Guidelines
       layout. Additionally, the circular logo and the
       name must always appear together.
     </p>
+    <p style="font-size: .7rem;">
+      * In rare occasions the horizontal logo is needed.
+      For these instances please contact <%= link_to "marketing@#{chef_domain}", "mailto:marketing@#{chef_domain}" %> for a request.
+    </p>
   </div>
   <div class="small-12 large-6 columns">
-    <div class="row" style="text-align:center;">
+    <div class="row example" style="text-align:center;">
```

Comment:
Is there an `example` class? And should we specify the text-alignment stuff somewhere in that instead (or a child of it), rather than repeat it on the `row` divs?